modules/remotecontrol/Receiver.js

```diff
@@ -0,0 +1,47 @@
+/* global APP, JitsiMeetJS */
+const ConferenceEvents = JitsiMeetJS.events.conference;
+
+/**
+ * This class represents the receiver party for a remote controller session.
+ * It handles "remote-control-event" events and sends them to the
+ * API module. From there the events can be received from wrapper application
+ * and executed.
+ */
+class Receiver {
```

Comment:
I like the idea. I'll thing about how to implement it. Maybe I can do the same for the controller and move the whole requesting permissions thing to the RemoteControl.js. WDYT?